src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -152,6 +163,10 @@ otbrError DBusThreadObjectRcp::Init(void)
                    std::bind(&DBusThreadObjectRcp::LeaveNetworkHandler, this, _1));
     RegisterMethod(OTBR_DBUS_THREAD_INTERFACE, OTBR_DBUS_SET_NAT64_ENABLED_METHOD,
                    std::bind(&DBusThreadObjectRcp::SetNat64Enabled, this, _1));
+    RegisterMethod(OTBR_DBUS_THREAD_INTERFACE, OTBR_DBUS_START_EPHEMERAL_KEY_MODE_METHOD,
```

Comment:
I don't think we need 2 different handlers here for start and stop. Can we write it as something like `SET_EPHEMERAL_KEY_MODE` and pass an argument for start/stop. For example, `DBusThreadObjectRcp::SetNat64Enabled` is implemented in this way.

Reply:
Use one dbus API with different parameters is one option. Choose to have separated start/stop APIs to have more straightforward control according to spec's requirement. Thanks.